qa/run-safe/valgrind_with_log_kill

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash -e
+# helper script for valgrind as root with kill on first error
+
+readonly myname=$(basename "$BASH_SOURCE")
+function myprint { printf "%s[Script %s]: %s %s\n" "$(tput setab 4)" "$myname" "$*" "$(tput sgr0)" ; }
```

Comment:
🚫 This time it's not needed imo, because it's 1-liner, and it's bit different between scripts (other colors)